roles/ssh_role/tasks/main.yml

```
@@ -0,0 +1,46 @@
+---
+# tasks file for ssh_role
+- name: Set authorized key on rpi taken from local file
+  ansible.posix.authorized_key:
+    user: "{{ ansible_user }}"
+    state: present
+    key: "{{ lookup('file', public_key_path) }}"
+
```

```suggestion

- name: create ssh directory
  file:
    path: ~/.ssh/
    state: directory

```
I did not have a ssh folder by defaut, thus it was failing ! (Might be a cleaner way to avoid this issue :) )

---

Oh my bad ! I did not think about this case, nice catch, thanks 👌

---

Thought the ssh folder is not in the RPI only if the RpiSETUP.md guidelines were not followed (because you need to created the .ssh)

But is it still cleaner to create the folder, I added this task